news/2019-05-10-spring-2019-fanart-contest.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Spring 2019 Fanart Contest"
+date: 2019-05-10 12:00:00 +0000
+---
+
+Better late than never! Spring might be on the way out, but talent is *always* in season, so whip out your drawing implement of choice and draw any of the osu! mascots enjoying Spring for fun and prizes!
+
+[![](https://assets.ppy.sh/contests/78/header.jpg)](https://osu.ppy.sh/community/contests/78)
+
+If you find yourself a person of artistic interest, this should be right up your alley! The artists among us have **14 days** to take a whirl at drawing any **spring-themed rendition of the osu! official or community mascots**.
+
+We mean spring-themed as in the season spring, not springs as in bouncy coils or whatever else.
+
+All entries that follow the rules laid out below will go up for 7 days of intense voting, and a selection of the highest-scoring winners will make it into the osu! game client as the Spring 2019 seasonal backgrounds. To celebrate, their authors will also receive **2 months of osu! supporter**. Nice!
+
+Want to enter? Read the rules below to get started, then [head on over to the contest listing page](https://osu.ppy.sh/community/contests/78) and submit your entry when you're ready. Oh, and by the way, **read the rules**. It's worth saying twice because we *always* have at least a dozen entries that never make it to voting for not following the rules!
+
+## Rules
+
+* **All entries MUST be OBVIOUSLY SPRING THEMED in some way.** Spring here refers to the season, not anything else. Flowers, sunshine, that sort of thing. Your work doesn't have to include flowers or sunshine specifically, but it does need to be "Spring" somehow. How you get there is entirely up to you, just make sure it's obvious!
```

Comment:
```suggestion
* **All entries must be obviously spring themed in some way.** Spring here refers to the season, not anything else. Flowers, sunshine, that sort of thing. Your work doesn't have to include flowers or sunshine specifically, but it does need to be "spring" somehow. How you get there is entirely up to you, just make sure it's obvious!
```
it's already the first rule, the double emphasis probably isn't necessary

Reply:
ok same with the others

Reply:
it's necessary, people STILL fuck it up even in caps and bold.